test/compflags/bradc/help/sub_test

```diff
@@ -1,5 +1,5 @@
-#!/bin/csh -f
+#!/bin/bash
 
-unsetenv CHPL_UNOPTIMIZED
+export -n CHPL_UNOPTIMIZED
```

Comment:
Huh, I think this is fine, but I would think `unset CHPL_UNOPTIMIZED` would be a closer equivalent